SDCTA/src/components/Body.tsx

```diff
@@ -0,0 +1,56 @@
+import landscape from '../Images/unsplash_dsV4yvL5UJk.jpg';
+import EdVis from '../Images/Graph.jpg';
+import './body.css'
+import EducationVis from '../Images/Education_Standin.png';
+import HomelessnessVis from '../Images/Homelessness_Standin.png';
+import MunicipalVis from '../Images/Municipal_Standin.png'
+
+
+export const Body = () => {
+	return (
+		<><div>
+			<div className="heading" >
+				SDTEF Data Museum
+			</div>
+			<div className="description" >
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
```

Comment:
Minor Point. Not sure how this alignment is showing up in vs code but you can use something like the Prettier [extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) . You can then enable Vscode to format with Prettier on [save](https://www.alphr.com/auto-format-vs-code/#:~:text=Click%20the%20%E2%80%9CSettings%E2%80%9D%20gear%20icon,%3A%20Format%20On%20Save%E2%80%9D%20option.)